lib/theme_data.dart

```diff
@@ -87,7 +87,7 @@ final ThemeData blueTheme = ThemeData(
         height: 1.28,
         fontFamily: 'IBMPlexSansMedium'),
     body1: TextStyle(
-        color: BreezColors.blue[900],
+        color: Color.fromRGBO(0, 133, 251, 1.0),
```

Comment:
Is this something requires going over all the UI? How can we make sure the body1 value is OK for other widgets? or if even used?

Reply:
Changed to 
`theme.themeId == "BLUE"
                                ? Color.fromRGBO(0, 133, 251, 1.0)
                                : Colors.white`